src/interactive_creative_file.js

```diff
@@ -1,8 +1,15 @@
-export function createInteractiveCreativeFile() {
+import { parserUtils } from './parser/parser_utils';
+
+export function createInteractiveCreativeFile(
+  interactiveCreativeAttributes = {}
```

Comment:
just a suggestion, you could destructure the object already here :)

Reply:
I think I am going to keep it as an object for now. Also for consistency with the creativeAttributes which are passed in the companionAd